documentation/wiki/ChangeWaves.md

```diff
@@ -57,13 +57,6 @@ A wave of features is set to "rotate out" (i.e. become standard functionality) t
 - [AnyHaveMetadataValue returns false when passed an empty list](https://github.com/dotnet/msbuild/pull/8603)
 - [Log item self-expansion](https://github.com/dotnet/msbuild/pull/8581)
 
-### 17.4
```

Comment:
We are also retiring 17.6 and 17.8 right? These are still included in the active changewaves here.

Reply:
Thanks!